Neos.ContentRepository.Core/Classes/Infrastructure/Property/PropertyType.php

```diff
@@ -38,9 +38,21 @@ final class PropertyType
 
     public const PATTERN_ARRAY_OF = '/array<[^>]+>/';
 
+    private ?self $arrayOfType;
```

Comment:
don't you have to initialize that to avoid exceptions when accessing it later?
```suggestion
    private ?self $arrayOfType = null;
```

Reply:
no i _want_ an exception as it should be only accessed internally if its `isArrayOf`, i know its a code-smell but well ^^

so it should be

```suggestion
    /** only set if {@see sef:: isArrayOf()} */
    private self $arrayOfType;
```